public/pages/CreateMonitor/components/MonitorDefinition/MonitorDefinition.js

```diff
@@ -19,6 +19,7 @@ import FormikSelect from '../../../../components/FormControls/FormikSelect/Formi
 const selectDefinitions = [
   { value: 'graph', text: 'Define using visual graph' },
   { value: 'query', text: 'Define using extraction query' },
+  { value: 'ad', text: 'Define using Anomaly detector' },
```

Comment:
This should not be selectable if AD is not installed.

Reply:
Current design will show this AD option. When use select this option, will show some warning message. We keep this option to let user know they can use anomaly detector's result as monitor's input. @mihirsoni , add more if we have other consideration to keep this option when AD plugin not installed.

![Screen Shot 2020-04-24 at 8 02 20 PM](https://user-images.githubusercontent.com/49084640/80269667-9272ba80-8666-11ea-9e2e-7f29c43ed54f.png)

Reply:
@ylwu-amzn @mihirsoni 

> We keep this option to let user know they can use anomaly detector's result as monitor's input.

This if fine when AD plugin is installed. But, if it's not installed then this should not be shown because they can't use this option. It's a negative experience in Alerting to have selectable options that can't be used.

Reply:
Sure, will discuss with UX designer to check why we have such design before.

Reply:
Confirm with UX designer, we should remove this option if AD not installed. Will fix this

Reply:
Done. 
![Screen Shot 2020-04-27 at 5 33 09 PM](https://user-images.githubusercontent.com/49084640/80433920-3352b700-88ad-11ea-855e-b10926a9bfbe.png)